README.md

```diff
@@ -198,3 +198,8 @@ You may have a mismatch between the embedded Android SDK Java and the JDK instal
 `
 
 Note that this change might cause problems with other Java-based applications (e.g., if you uninstall Android Studio).
+
+#### gradlew Failure: `java.lang.NullPointerException (no error message).`
+If you encounter the `java.lang.NullPointerException (no error message).` when running `gradlew`, please make sure your Java version for this project is Java 8. You can reinstall Java or set a specific JDK version for the project like this:
```

Comment:
Thank you, @huangyz0918, for troubleshooting this problem and adding it to the README!

How about we just stop after "this project is Java 8" and not provide specific guidance. I'm suggesting this because different platforms will have different ways of getting the JDK location and generally, changing the Android Studio JDK location won't change `gradlew` behavior. I think it's enough to tell developers what the problem is and then let them figure out their own solution or ask in #collect-code.